frontend/src/pages/Dashboard/index.tsx

```diff
@@ -88,54 +128,62 @@ const Dashboard: React.FC = () => {
+                    <UserGroupCard
+                      label="Students"
+                      givenUsersList={classroomUsersList.filter(
                         (user) => user.classroom_role === "STUDENT"
-                      )
-                    )
-                  }
-                />
+                      )}
+                      onClick={() =>
+                        handleUserGroupClick(
+                          "Student",
+                          classroomUsersList.filter(
+                            (user) => user.classroom_role === "STUDENT"
+                          )
+                        )
+                      }
+                    />
 
-                <UserGroupCard
-                  label="TAs"
-                  givenUsersList={classroomUsersList.filter(
-                    (user) => user.classroom_role === "TA"
-                  )}
-                  onClick={() =>
-                    handleUserGroupClick(
-                      "TA",
-                      classroomUsersList.filter(
+                    <UserGroupCard
+                      label="TAs"
+                      givenUsersList={classroomUsersList.filter(
                         (user) => user.classroom_role === "TA"
-                      )
-                    )
-                  }
-                />
+                      )}
+                      onClick={() =>
+                        handleUserGroupClick(
+                          "TA",
+                          classroomUsersList.filter(
+                            (user) => user.classroom_role === "TA"
+                          )
+                        )
+                      }
+                    />
 
-                <UserGroupCard
-                  label="Professors"
-                  givenUsersList={classroomUsersList.filter(
-                    (user) => user.classroom_role === "PROFESSOR"
-                  )}
-                  onClick={() =>
-                    handleUserGroupClick(
-                      "Professor",
-                      classroomUsersList.filter(
+                    <UserGroupCard
+                      label="Professors"
+                      givenUsersList={classroomUsersList.filter(
                         (user) => user.classroom_role === "PROFESSOR"
-                      )
-                    )
-                  }
-                />
+                      )}
+                      onClick={() =>
+                        handleUserGroupClick(
+                          "Professor",
+                          classroomUsersList.filter(
+                            (user) => user.classroom_role === "PROFESSOR"
+                          )
+                        )
+                      }
+                    />
+                  </div>
+
+                  <SimpleMetric metricTitle="Created on" metricValue={formatDate(selectedClassroom.created_at ?? null)}></SimpleMetric>
```

Comment:
I see "Dec 31, 1" from this which definitely is not right

Reply:
Huh that's odd, can you attach a ss? I'm using a function you'd defined in utils and it works correctly everywhere else

Reply:
done